BUILDING.md

```diff
@@ -270,6 +270,24 @@ This will generate Filament's Android binaries in `out/android-release`. This lo
 to build the Android Studio projects located in `filament/android`. After install, the library
 binaries should be found in `out/android-release/filament/lib/arm64-v8a`.
 
+#### Adreno workaround
+Adreno GPUs have issues using the optimized shaders compiled from Filament's materials. To
```

Comment:
Adrenos represent a large chunk of the population, we should really find a better workaround than asking devs to turn off optimizations entirely.

Reply:
We tried!

Reply:
Then we should at least say it's specific to Vulkan?

Reply:
no longer required as it's separately addressed in #6757